manifests/config.pp

```diff
@@ -10,13 +10,87 @@
     system => true,
   }
 
+  case $::osfamily {
+    'RedHat':  {
+      $nologin_shell = '/sbin/nologin'
+      case $::operatingsystemmajrelease {
+        '7': {
+          file {'/etc/systemd/system/caddy.service':
```

Comment:
instead of using a file resource, use [systemd::unit_file](https://github.com/camptocamp/puppet-systemd#unit-files). Please add the module to the metadata.json

Reply:
Fixed in 5e1e72f305c530ee8c52972eace9e0a1b8dcdced